multi-agent/api/flask/endpoints/resources.py

```
@@ -1,238 +1,239 @@
+    max_workers = min(max_workers, 20)
+    
+    try:
+        results = svc.validate_resources(
+            rids=resource_ids,
+            timeout_seconds=timeout_seconds,
+            max_workers=max_workers,
+        )
+        return jsonify([r.to_dict() for r in results]), 200
+    except RuntimeError as e:
+        return jsonify({"error": str(e)}), 500
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)